docs/deprecated/clusters/public-cloud/azure/required-permissions.md

```diff
@@ -0,0 +1,967 @@
+          "Microsoft.Storage/storageAccounts/blobServices/containers/write",
+          "Microsoft.Storage/storageAccounts/read",
+          "Microsoft.Storage/storageAccounts/write"
+        ],
+        "NotActions": [],
+        "AssignableScopes": ["/subscriptions/$SUBSCRIPTION_ID"]
+      }
+      ```
+    ```json
+    {
+      "Name": "Static Placement Palette Deployer - Compute Gallery Scope",
+      "IsCustom": true,
+      "Description": "Can deploy the compute gallery level components of Azure IaaS clusters using static placement with Palette.",
+      "Actions": [
+        "Microsoft.Compute/galleries/images/read", 
+        "Microsoft.Compute/galleries/images/versions/read"
+        ],
+      "NotActions": [],
+      "AssignableScopes": ["/subscriptions/$SUBSCRIPTION_ID"]
+    }
+    ```
+
+    ```json
+    {
+      "Name": "Static Placement Palette Deployer - Subnet Scope",
+      "IsCustom": true,
+      "Description": "Can deploy the subnet level components of Azure IaaS clusters using static placement with Palette.",
+      "Actions": [
+        "Microsoft.Network/virtualNetworks/subnets/join/action",
+        "Microsoft.Network/virtualNetworks/subnets/read"
+      ],
+      "NotActions": [],
+      "AssignableScopes": ["/subscriptions/$SUBSCRIPTION_ID"]
+    }
+    ```
+
+    ```json
+    {
+      "Name": "Static Placement Palette Deployer - Virtual Network Scope",
+      "IsCustom": true,
+      "Description": "Can deploy the virtual network level components of Azure IaaS clusters using static placement with Palette.",
+      "Actions": ["Microsoft.Network/virtualNetworks/read"],
+      "NotActions": [],
+      "AssignableScopes": ["/subscriptions/$SUBSCRIPTION_ID"]
+    }
+    ```
+
+    </details>
+
+6.  Create a role for each of the JSON files you created in the previous steps. Issue the following commands to create
+    the roles.
+
+    ```shell
+    az role definition create --role-definition @static-permissions.json --output table
+    az role definition create --role-definition @static-gallery-scope-permissions.json --output table
+    az role definition create --role-definition @static-subnet-scope-permissions.json --output table
+    az role definition create --role-definition @static-virtual-network-scope-permissions.json --output table
+    ```
+
+7.  Export resource group, virtual network, subnet, and compute gallery names you desire Palette to use to a set of
+    variables.
+
+    ```shell
+    export RESOURCE_GROUP_NAME=<resource-group-name>
+    export VIRTUAL_NETWORK_NAME=<virtual-network-name>
+    export SUBNET_NAME=<subnet-name>
+    export COMPUTE_GALLERY_NAME=<compute-gallery-name>
+    ```
+
+8.  Export the
+    [security principal](https://learn.microsoft.com/en-us/azure/role-based-access-control/overview#security-principal)
+    object ID you want to use with Palette to a variable.
+
+    ```shell
+    export ASSIGNEE="<security_principal_object_id>"
```

Comment:
this needs to say `<security_principal_client_id>` instead of `<security_principal_object_id>`